### PR TITLE
Create nameof.md

### LINKED ISSUE #9690: Missing NameOf in Visual Basic Language Reference
The NameOf keyword is missing in the keyword page, and according to [#366](https://github.com/dotnet/vblang/issues/366), the keyword is missing in the language reference.

The Visual Basic What's New topic should also be updated to link to this topic for detailed information.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bb2d0d24-a228-ab33-1c62-3b0a701ff318
* Version Independent ID: 9ef3c275-ce7b-808b-8acd-3d394296d897
* Content: [Keywords (Visual Basic)](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/keywords/)
* Content Source: [docs/visual-basic/language-reference/keywords/index.md](https://github.com/dotnet/docs/blob/master/docs/visual-basic/language-reference/keywords/index.md)
* Product: **dotnet-visualbasic**
* GitHub Login: @KathleenDollard
* Microsoft Alias: **kdollard**

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #9690

### CODE REVIEW

**Comment by Youssef1313** [comment]
@mairaw, A link to the created NameOf.md should be added in `operators/index.md`.
Is there any easy way to add it to the right cell in the table ?
The table is ordered alphabetically so I can't just place it at the end.

**Review by BillWagner — APPROVED** [review]
This LGTM.
I'll :shipit: now.

**Comment by Youssef1313** [comment]
> 
> 
> @mairaw, A link to the created NameOf.md should be added in `keywords/index.md`.
> Is there any easy way to add it to the right cell in the table ?
> The table is ordered alphabetically so I can't just place it at the end.

@BillWagner, this should be addressed in another PR, but working with such a table manually seemed hard a little bit for me 😕 .

**Comment by mairaw** [comment]
I don't think there's an easy way unfortunately 😢 maybe some external tools that I'm not aware of could do this.

**Comment by mairaw** [comment]
Do you want to file a new issue about the missing keyword on the index page?

**Comment by Youssef1313** [comment]
@mairaw, I've made that manually and opened #15500. Hope I didn't mess up things in it 😄 .

File: docs/visual-basic/language-reference/operators/nameof.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "NameOf operator - Visual Basic"
```
Comment (gewarren) [inline docs/visual-basic/language-reference/operators/nameof.md]: Oops this wasn't added to the TOC - I think all new topics need to go into a TOC.

**Pull request merged**